x-pack/plugins/security_solution/server/lib/detection_engine/prebuilt_rules/api/review_rule_installation/review_rule_installation_route.ts

```diff
@@ -48,7 +47,7 @@ export const reviewRuleInstallationRoute = (router: SecuritySolutionPluginRouter
 
         const body: ReviewRuleInstallationResponseBody = {
           stats: calculateRuleStats(installableRules),
-          rules: calculateRuleInfos(installableRules),
+          rules: installableRules.map(convertPrebuiltRuleAssetToRuleResponse),
```

Comment:
It is better to avoid using functions as callbacks if they were not designed for that. Much safer would be to write:

```ts
installableRules.map(rule => convertPrebuiltRuleAssetToRuleResponse(rule))
```